docs/developing_with_spectralcube.rst

```diff
@@ -0,0 +1,93 @@
+.. _doc_developersnotes:
+
+Notes for development using spectral-cube
+=========================================
+.. currentmodule:: spectral_cube
```

Comment:
whooaaaa didn't know you could do this.

Reply:
Me either! I copy-pasted from docs Chris wrote.